### ISSUE TITLE
upstream incompatibility between python regex library 2022.3.15 and dateparser breaks API

### ISSUE THREAD

**Issue by mmguero** [idaholab/Malcolm#84]
In the last day or two version `2022.3.15` of the python [regex](https://pypi.org/project/regex/#history) library was put out. [`dateparser`](https://pypi.org/project/dateparser/) depends on that library, but some sort of incompatibility was introduced in this new version. 

When calling /mapi/agg, the error `bad escape \d at position 7` is now returned instead.

A few stackoverflow threads ([1](https://stackoverflow.com/questions/71498132/error-in-heroku-regex-regex-core-error-bad-escape-d-at-position-7-when-usin), [2](https://stackoverflow.com/questions/71502356/python-binance-client-get-historical-klines-error), [3](https://stackoverflow.com/questions/71496687/dateparser-throws-regex-regex-core-error)) seem to have come to the same conclusion.

In the meantime I'm going to pin `regex` to `2022.3.2` in the API docker container.

I don't know if this made it into the v5.2.8 release yesterday. If it did, I'll have to put out a patch fix for it as it renders the API useless.

**Comment by mmguero** [comment]
Well, crap:

```
› docker run --rm --entrypoint=/bin/bash malcolmnetsec/api:5.2.8 -c "pip freeze 2>/dev/null | grep regex"
regex==2022.3.15
```

**Comment by mmguero** [comment]
Upstream issue information:

`dateparser` issue: https://github.com/scrapinghub/dateparser/issues/1045

breaking commit in `regex` library: https://github.com/mrabarnett/mrab-regex/commit/138970bafb3d6fbe0987632ee149c04e8b5acf95

**Comment by asadurski** [comment]
Hello, this is resolved in [dateparser version 1.1.1](https://github.com/scrapinghub/dateparser/releases/tag/v1.1.1), released yesterday.

**Comment by mmguero** [comment]
> Hello, this is resolved in [dateparser version 1.1.1](https://github.com/scrapinghub/dateparser/releases/tag/v1.1.1), released yesterday.

Thanks for the reply and the great library.
